ush/config.sh.RRFS_CONUS_13km_ens

```diff
@@ -115,28 +116,27 @@ PREEXISTING_DIR_METHOD="upgrade" # "rename"
 INITIAL_CYCLEDEF="${DATE_FIRST_CYCL}1800 ${DATE_LAST_CYCL}2300 24:00:00"
 BOUNDARY_CYCLEDEF="${DATE_FIRST_CYCL}1800 ${DATE_LAST_CYCL}0000 06:00:00"
 PROD_CYCLEDEF="00 00,18-23 07 02 2022 *"
+PROD_CYCLEDEF="00 01,02,04,05,07,08,10,11,13,14,16,17,19,20,22,23 ${CYCLEDAY} ${CYCLEMONTH} 2022 *"
 ENSFCST_CYCLEDEF="00 00 08 02 2022 *"
-#RECENTER_CYCLEDEF="00 19 * 10 2022 *"
-POSTPROC_CYCLEDEF="${DATE_FIRST_CYCL}1800 ${DATE_LAST_CYCL}0000 01:00:00"
-POSTPROC_LONG_CYCLEDEF="00 00 08 02 2022 *"
 ARCHIVE_CYCLEDEF="${DATE_FIRST_CYCL}1500 ${DATE_LAST_CYCL}2300 24:00:00"
-#ARCHIVE_CYCLEDEF="${DATE_FIRST_CYCL}0700 ${DATE_LAST_CYCL}2300 24:00:00"
 if [[ $DO_SPINUP == "TRUE" ]] ; then
   SPINUP_CYCLEDEF="${DATE_FIRST_CYCL}1800 ${DATE_LAST_CYCL}2300 24:00:00"
 fi
 
 FCST_LEN_HRS="1"
 FCST_LEN_HRS_SPINUP="1"
-FCST_LEN_HRS_ENSFCST="24"
+FCST_LEN_HRS_ENSFCST="1"
```

Comment:
This is the default value - 1.
We only need to set this for "DO_ENSFCST", ensemble free forecast

Reply:
deleted FCST_LEN_HRS_ENSFCST